full-service/src/db/transaction_log.rs

```diff
@@ -777,16 +777,16 @@ impl TransactionLogModel for TransactionLog {
     }
 
     fn update_pending_associated_with_txo_to_succeeded(
-        txo_id_hex: &str,
+        txo_out_id_hex: &str,
```

Comment:
🤷 not sure on the addition of `out` here. It's nicer to have it in the signature than to look at the implementation to see the `transaction_output_txos`

Reply:
Given the reason for the PR, it makes sense to be more clear that this function should only be used for a `transaction_output` `txo`.

What do you think of keeping consistent with that notation, which is used already used -- prefixing with `transaction_output` -- i.e. `transaction_output_txo_id_hex`? 

I don't have a strong opinion here, but I prefer avoiding making new terminology when there is an existing pattern, and especially putting words in the middle of existing patterns, such as `txo_id` and `txo_id_hex`.

Reply:
The main problem I see is it comes across as redundant
`txo` stands for transaction output so to have `transaction_output_txo_id_hex` is communicating `transaction_output_transaction_output_id_hex`

Though the table is named `transaction_output_txos`... so it's probably not inconsistent, i'll go ahead and make the change

Reply:
done in [ebd3a46](https://github.com/mobilecoinofficial/full-service/pull/1027/commits/ebd3a4617a647ebcec4c1d655e4d80205179fc7e)